pkgs/development/python-modules/testtools/default.nix

```diff
@@ -0,0 +1,38 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pbr
+, python_mimeparse
+, extras
+, lxml
+, unittest2
+, traceback2
+, isPy3k
+}:
+
+buildPythonPackage rec {
+  pname = "testtools";
+  version = "1.8.0";
+  name = "${pname}-${version}";
+
+  # Python 2 only judging from SyntaxError
+#   disabled = isPy3k;
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "15yxz8d70iy1b1x6gd7spvblq0mjxjardl4vnaqasxafzc069zca";
+  };
+
+  propagatedBuildInputs = [ pbr python_mimeparse extras lxml unittest2 ];
```

Comment:
@FRidh Do you know why this depends on lxml? I can't find it in https://github.com/testing-cabal/testtools/blob/b4d158ab8aff96c4212d49b3f0d2c3001682c3c9/requirements.txt

Reply:
No idea, it was there already. Let's throw it out.

Reply:
I'll try enabling the tests as well